core/modules/qdisp/QueryRequest.cc

```diff
@@ -476,8 +483,7 @@ void QueryRequest::_processData(JobQuery::Ptr const& jq, int blen, bool xrdLast)
     }
 
     // Read the next header
-    largeResult = false;
-    nextBufSize = 0;
+    // Values for largeResult and nextBufSize will be filled in by flush().
```

Comment:
Does this apply to `last` as well? If so then it should be mentioned here as well.